general_autoencoder.py

```diff
@@ -316,11 +327,31 @@ def _fit_from_processed_data(self,
                 self.Z = self.encode(self.X)
                 
             self.Xr = self.decode(self.Z)
-            self.combined_loss, self.binary_loss, self.continuous_loss, self.reg_loss = self.get_loss()
+            self.combined_loss, self.binary_loss, self.continuous_loss, self.reg_loss = self.get_loss(self.X, self.Xr)
```

Comment:
are self.X and self.Xr sufficient to compute the loss in all cases? won't we also need, eg, self.ages in some cases?